src/content/blog/misleading-promises-of-the-worlds-fastest-anonymous-military-grade-vpns.md

```diff
@@ -0,0 +1,162 @@
+
+## 4\. “Military grade encryption”
+
+or "industry-leading encryption" and "most encryption"
+
+“Military grade encryption” is a popular marketing gimmick in the VPN provider sales vocabulary. There is no fixed standard set in militaries for encryption, and implementations vary across different segments of armed forces.
+
+There are weak encryption protocols which you obviously don’t want to see used by a VPN service e.g. PPTP. However, the vast majority of providers implement the same level of encryption using OpenVPN or Wireguard with the default cipher (AES-256-GCM / ChaCha20). Providers don’t develop their own encryption protocols (excluding obfuscation layers). Providers calling their encryption technology “industry-leading” is misleading.
+
+Misleading samples:  
+ExpressVPN Blog - "ExpressVPN for routers protects all your devices with military grade encryption!"  
+Private Internet Access home page - "...to provide the highest speeds and most encryption."  
+SurfShark streaming promo - "all of your data is protected by a military-grade encryption system"
+
+![](/images-static/uploads/Screenshot-2020-12-01-at-16.46.51.png)
+
+## 5\. “The best VPN”
+
+or "market leading VPN" and “best VPN for X”
+
+Eight out of ten VPN providers we looked at claimed they are the best for everyone or for a specific purpose. This number alone demonstrates the issue with this claim.
+
+No VPN solution works universally well for each customer’s needs. A comparison website, after careful, independent evaluation might judge that a brand offers the “best all-around solution on the market”. But for service providers to claim they are the best for you is a bold move.
+
+This problem points to a general issue with US-focused marketing, where advertisers enjoy flexibility for wording in advertising. But even there you need to support your claims with substantial, objective evidence.
+
+Misleading samples:  
+TorGuard country promo pages - "Don’t settle for second best. Use the best VPN for Australia."  
+NordVPN home page title - "NordVPN: Best VPN service. Online security starts with a click."  
+VyprVPN home page - "Get the best VPN for streaming with lightning-fast and reliable connections"
+
+![](/images-static/uploads/Screenshot-2020-12-01-at-16.45.31.png)
+
+* * *
+
+We have empathy for the marketing teams of VPN providers. The competition is fierce. Writing copy that sells is hard. You need to optimise for juicy search keywords.
+
+Yet, you should not make promises or claims that are untrue.
+
+Start removing the misleading words from your websites today.
+
+* * *
+
+_Addendum:_
+
+VPN providers included in this research and their score:  
+Private Internet Access 5/5  
+CyberGhost 5/5  
+HotSpotShield 5/5  
+TorGuard 4/5  
+NordVPN 4/5  
+ExpressVPN 4/5  
+VyprVPN 3/5  
+SurfShark 3/5  
+ProtonVPN 1/5  
+TunnelBear 0/5
+
+[Download](/resources/misleading_overpromise.xlsx) the full table to review scores
```

Comment:
punctuation missing, please add '.' at the end.

Reply:
Fixed.
I'm merging to `staging`.